src/middleware/persist.ts

```diff
@@ -66,12 +64,12 @@ export type PersistOptions<S, PersistedState = Partial<S>> = {
    * A function to perform persisted state migration.
    * This function will be called when persisted state versions mismatch with the one specified here.
    */
-  migrate?: (persistedState: any, version: number) => S | Promise<S>
+  migrate?: (persistedState: unknown, version: number) => S | Promise<S>
```

Comment:
`unknown` is preferred over `any`.

Reply:
Is it possible that `unkown` causes some type errors when trying to access `persistedState`'s properties?

Reply:
Yes it does, that's the point of the change. `persistedState` comes from the (local)storage so it can be anything (hence `unknown`) the users should responsibly handle it.